loki-logback-appender/src/test/java/com/github/loki4j/logback/integration/FastSendTest.java

```diff
@@ -78,4 +79,18 @@ public void testJavaProtobufFastSend() throws Exception {
         client.testHttpSend(label, events, appender, jsonEncoder(false, label));
     }
 
+    @Test
+    @Category({IntegrationTests.class})
+    public void testJavaJsonFastSendWithTenant() throws Exception {
+        var label = "testJavaJsonFastSendWithTenant";
+        var encoder = protobufEncoder(false, label);
+        var sender = javaHttpSender(urlPush);
+        sender.setTenantId(tenant);
+        var appender = appender(10, 1000, encoder, sender);
+
+        var events = generateEvents(1000, 10);
+        client.testHttpSend(label, events, appender, jsonEncoder(false, label));
```

Comment:
Is this tenant really applied in Loki? I thought it should require changes in receiving part as well.

Reply:
that requires that loki starts in multi-tenant mode. I suppose this is currently is not configured on server used by integration tests. Can you have some server running in multi-tenant mode ?

Reply:
I suppose in single mode that header is just ignored